### ISSUE TITLE
pointnet2 installation failed

### ISSUE THREAD

**Issue by Leerw** [facebookresearch/votenet#64]
After I succeed running `python setup.py install`, I failed to run `python models/votenet.py` as follows:
```
Traceback (most recent call last):
  File "/mnt/8T/leerw/code/votenet/pointnet2/pointnet2_utils.py", line 26, in <module>
    import pointnet2._ext as _ext
ImportError: dynamic module does not define module export function (PyInit__ext)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "models/votenet.py", line 19, in <module>
    from backbone_module import Pointnet2Backbone
  File "/mnt/8T/leerw/code/votenet/models/backbone_module.py", line 19, in <module>
    from pointnet2_modules import PointnetSAModuleVotes, PointnetFPModule
  File "/mnt/8T/leerw/code/votenet/pointnet2/pointnet2_modules.py", line 21, in <module>
    import pointnet2_utils
  File "/mnt/8T/leerw/code/votenet/pointnet2/pointnet2_utils.py", line 30, in <module>
    "Could not import _ext module.\n"
ImportError: Could not import _ext module.
Please see the setup instructions in the README: https://github.com/erikwijmans/Pointnet2_PyTorch/blob/master/README.rst

```

I can `import torch; import pointnet2` without errors, but not when `import torch; import pointnet2._ext as _ext`

**Comment by StevenZzz07** [comment]
Hi! I am faced with exactly the same issue as yours. Installing PyTorch and Tensorflow, then running `python setup.py install`. When it comes to the next steps, the same bugs come out: "Could not import _ext module.\n". Have you solved this problems? Maybe we can discuss about it.

**Comment by StevenZzz07** [comment]
Hello, i have solved out this problem. I used another platform with GPU 1660ti and CUDA10.0, then the problem disappear. So the bug may be related to the CUDA version i guess.

**Comment by Leerw** [comment]
@StevenZzz07 Yes, I solved this issue, I have tensorflow-gpu==1.14 and pytorch==1.1.0 and CUDA 10.0 installed, I fixed it by install original Pointnet_Pytorch code the repo borrowed from and set the PYTHONPATH=`folder of Pointnet_Pytorch`. But I still do not know what is wrong with `python setup.py install`

**Comment by phy12321** [comment]
I met this error too. The problem in my case is that the python i  use in command is system python,which locate in /usr/local/bin/... ,but i run my code in conda env. so i ran this :
`/home/anaconda/envs/envs_name/bin/python setup.py install `
then check if pointnet2 is installed in my conda envs:
`conda list`
![image](https://user-images.githubusercontent.com/25128870/70965379-74021e80-20ca-11ea-8644-335663515c95.png)
and run :
`python models/votenet.py`
it works!

**Comment by mojoee** [comment]
For me, the problem was solved by installing the correct versions of tensorflow(1.14) and pytorch (1.1). However, another problem popped up afterwards.

**Comment by ch-sa** [comment]
For me the problem vanished after upgrading from `python 3.6.8` to `python 3.7.7`.

**Comment by bigeyesung** [comment]
I have a similar problem. Solved by install CUDA 10.0. tensorflow1.14, pytorch 1.2
